fluentlenium-spock/src/main/groovy/org/fluentlenium/adapter/spock/FluentSpecification.groovy

```diff
@@ -0,0 +1,56 @@
+package org.fluentlenium.adapter.spock
+
+import org.fluentlenium.adapter.FluentTestRunnerAdapter
+import org.fluentlenium.adapter.junit.FluentTestRule
+import org.junit.ClassRule
+import org.junit.Rule
+import org.junit.rules.TestRule
+import org.junit.runner.Description
+import org.junit.runners.model.Statement
+import spock.lang.Specification
+
+class FluentSpecification extends Specification {
+
+    protected FluentSpecificationAdapter fl = new FluentSpecificationAdapter()
```

Comment:
Well ... I even think this variable should be private and annotated with @Delegate (Lombok) for this to simulate the multiple inheritance, so user could then write this : 

```groovy
@Wait
class FluentleniumSpec extends FluentSpecification {

    def "Title of duck duck go"() {
        given:
        goTo("https://duckduckgo.com")

        when:
        $("#search_form_input_homepage").fill().with("FluentLenium")
        $("#search_button_homepage").submit()

        then:
        window().title().contains("FluentLenium")
    }
}
```